ql/src/test/results/clientpositive/llap/lvj_mapjoin.q.out

```diff
@@ -121,7 +121,6 @@ STAGE PLANS:
                 TableScan
                   alias: expod1
                   filterExpr: aid is not null (type: boolean)
-                  probeDecodeDetails: cacheKey:HASH_MAP_MAPJOIN_39_container, bigKeyColName:aid, smallTablePos:1, keyRatio:1.0
```

Comment:
I didn't understand why this part of the plan changed.

Reply:
The probe decode optimization relies on the presence of semijoins (https://github.com/apache/hive/blob/5f57814ed743a411c8fa7c647c24c98461271fe3/ql/src/java/org/apache/hadoop/hive/ql/parse/TezCompiler.java#L1401). Semijoins depend on the `SyntheticJoinPredicate` transformation thus probe decode depends transitively on `SyntheticJoinPredicate`. 

This PR disables `SyntheticJoinPredicate` transformation for branches with lateral views (present in `lvj_mapjoin.q` test) thus semijoins are not considered and neither probe decode.